build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("local-backend"))
```

Comment:
> I meant to have some dir which holds the officially supported backends 

I was thinking that dir could be the current `backend` dir but that won't work, I'm gonna try with another `default-backends` or something